doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -1,5 +1,3 @@
-:orphan:
-
 .. _ug_nrf54h20_custom_pcb:
```

Comment:
@KAGA164 @pdunaj @ns-tolu @SylwesterKonczyk @nordic-krch Could you review this document? Is this still relevant? Should it be orphaned for this release ?